interface.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import importlib
+import os
+import sys
+from datetime import datetime, timedelta
+
+import pandas as pd
+import pytz
+from sqlalchemy import create_engine
+
+raw_data_tables = [
+    'applications_foreground',
+    'battery',
+    'calls',
+    'light',
+    'locations',
+    'messages',
+    'screen'
+]
+
+
+def read_data_func(datastore, container):
+    if datastore == 'sql':
```

Comment:
We should probably move all the raw strings into "consts" file somewhere. I'm not sure what the best practice is for constant handling in Python, but something like constants.SQL_DATASTORE and constants.CSV_DATASTORE here, in a separate constants.py file?

Reply:
That's what we do in DLATK. We have a file called `dlaConstands.py`, import via

`import dlaConstants as dlac`

then bring in specific variables `dlac.SOME_CONSTANT`. We also have the ability to override any of these constants via a command line flag which reads an `.ini` file.

Reply:
We want to further abstract this so that we have system wide constants in a Python file, install level constants in some other text file and then user level constants in the `ini` file. 

This is done so that users don't have to go into the Python to make changes. If you use `pip` to install then the Python files might go to some hard to find directories. Plus if there is one install on the server and multiple people using the same installation, then you might want different users to have different settings.